x/ibc/client/cli/cli.go

```diff
@@ -0,0 +1,43 @@
+package cli
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	ics02 "github.com/cosmos/cosmos-sdk/x/ibc/02-client"
+	"github.com/cosmos/cosmos-sdk/x/ibc/types"
+)
+
+// GetTxCmd returns the transaction commands for this module
+func GetTxCmd(storeKey string, cdc *codec.Codec) *cobra.Command {
+	ibcTxCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "IBC transaction subcommands",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
```

Comment:
I don't think we need this `client.ValidateCmd`. Can anyone change my mind?

Reply:
this is the case for other modules as well. Mind opening an issue for it?